src/__snapshots__/App.test.js.snap

```diff
@@ -0,0 +1,1367 @@
+    className="overlay"
+    style={
+      Object {
+        "width": 500,
+      }
+    }
+  >
+    <h2>
+      Fill the form
+    </h2>
+    <label
+      className="label"
+    >
+      Label:
+      <input
+        onChange={[Function]}
+        type="text"
+        value=" "
+      />
+    </label>
+    <div
+      className="inputForm"
+    >
+      <div>
+        Type
+        <div
+          className="inputForm"
+        >
+          <input
+            checked={false}
+            id="text"
+            name="type"
+            onChange={[Function]}
+            type="radio"
+            value="text"
+          />
+          <label
+            htmlFor="text"
+          >
+            Textfiled
+          </label>
+        </div>
+        <div
+          className="inputForm"
+        >
+          <input
+            checked={false}
+            id="checkbox"
+            name="type"
+            onChange={[Function]}
+            type="radio"
+            value="checkbox"
+          />
+          <label
+            htmlFor="checkbox"
+          >
+            CheckBox
+          </label>
+        </div>
+        <div
+          className="inputForm"
+        >
+          <input
+            checked={false}
+            id="select"
+            name="type"
+            onChange={[Function]}
+            type="radio"
+            value="select"
+          />
+          <label
+            htmlFor="select"
+          >
+            Select
+          </label>
+        </div>
+        <div
+          className="inputForm"
+        >
+          <input
+            checked={false}
+            id="radios"
+            name="type"
+            onChange={[Function]}
+            type="radio"
+            value="radios"
+          />
+          <label
+            htmlFor="radios"
+          >
+            Radios
+          </label>
+        </div>
+      </div>
+      <div>
+        <label>
+          ADD MORE OPTIONS
+        </label>
+        <form
+          className="options-form"
+        >
+          <button
+            className="button-option"
+            onClick={[Function]}
+          >
+            ADD
+          </button>
+        </form>
+      </div>
+    </div>
+    <textarea
+      className="text-area"
+      onChange={[Function]}
+      value="{\\"label\\":\\" \\",\\"type\\":\\"select\\",\\"options\\":[]}"
+    />
+    <button
+      className="button"
+      onClick={[Function]}
+    >
+      Save
+    </button>
+    <button
+      className="button"
+      onClick={[Function]}
+    >
+      Cancel
+    </button>
+  </div>
+</div>
+`;
+
+exports[`Advanced tests of app  Selecting the type of radio buttons (select), add fields and input values then click save button 1`] = `
+<div>
+  <div
+    className="input-conntainer"
+  >
+    <input
+      className="input-field"
+      onClick={[Function]}
+      placeholder="Click here to for input text"
+      readOnly={true}
+      type="text"
+      value={
+        Object {
```

Comment:
Not sure if this is a problem with the snapshot or if it is really like this.
But is the text shown in the input field really: `Object { ... }`?

Reply:
No it's not shown in input field, only in snapshot.